src/workerd/api/actor-state.h

```diff
@@ -386,13 +408,28 @@ class DurableObjectState: public jsg::Object {
   // If no tag is provided, an array of all accepted WebSockets is returned.
   // Disconnected WebSockets are automatically removed from the list.
 
+  // void setWebSocketAutoresponse(kj::String request, kj::String response);
+  void setWebSocketAutoresponse(jsg::Optional<jsg::Ref<api::WebSocketRequestResponsePair>> maybeReqResp);
+  // Sets an object-wide websocket auto response message for a specific
+  // request string. All websockets belonging to the same object mustkj::Maybe<kj::Date> getAutoResponseTimestamp();
+  // reply to request with response and store the request recived timestamp.
```

Comment:
"reply to the request with the matching response, then store the timestamp at which the request was received".